diff -rupN SickBeard-German-004cb40.orig/sickbeard/germandates.py SickBeard-German-004cb40/sickbeard/germandates.py
--- SickBeard-German-004cb40.orig/sickbeard/germandates.py	2013-02-09 17:18:49.440660093 -0500
+++ SickBeard-German-004cb40/sickbeard/germandates.py	2013-02-09 17:41:45.450828872 -0500
@@ -8,12 +8,12 @@ from sqlalchemy.orm import sessionmaker,
 from sickbeard import logger
 from datetime import datetime, timedelta
 import time
+import sickbeard
 
 ##define a database and add stuff to it
 #TODO: Rewrite with mappers!
 #USE SCOPED SESSIONS ...
 
-engine = create_engine('sqlite:///germandates.db', echo=False)
 Base = declarative_base()
 
 
@@ -82,11 +82,12 @@ class Version(Base):
 	def __repr__(self):
 		return u"<Version('next_update:{0}>".format(self.nextupdate)
 
-
-Base.metadata.create_all(engine)
-Session = sessionmaker()
-Session.configure(bind=engine)
-session = scoped_session(Session)
+def init():
+	engine = create_engine('sqlite:///'+ sickbeard.DATA_DIR+'/germandates.db', echo=False)
+	Base.metadata.create_all(engine)
+	Session = sessionmaker()
+	Session.configure(bind=engine)
+	session = scoped_session(Session)
 
 def requestSlug(tvdbid=None, showname=None):
 	if tvdbid:
diff -rupN SickBeard-German-004cb40.orig/sickbeard/__init__.py SickBeard-German-004cb40/sickbeard/__init__.py
--- SickBeard-German-004cb40.orig/sickbeard/__init__.py	2013-02-09 17:18:49.440660093 -0500
+++ SickBeard-German-004cb40/sickbeard/__init__.py	2013-02-09 17:21:22.530037161 -0500
@@ -827,7 +827,7 @@ def initialize(consoleLogging=True):
 
         showList = []
         loadingShowList = {}
-
+	germandates.init()
         __INITIALIZED__ = True
         return True
 
